test/js/bun/test/test-test.test.ts

```diff
@@ -2612,3 +2612,51 @@ it("should return non-zero exit code for invalid syntax", async () => {
     await rm(test_dir, { force: true, recursive: true });
   }
 });
+
+describe("skip test inner", () => {
+  it("should pass", () => {
+    expect(2 + 2).toBe(4);
+  });
+
+  describe.skip("should skip", () => {
+    it("should throw", () => {
+      throw new Error("This should not throw. `.skip` is broken");
+    });
+  });
+});
+
+describe.skip("skip test outer", () => {
```

Comment:
can you add a test that checks we also skip `beforeAll`, `afterAll`, `beforeEach`, and `afterEach` (assuming Jest does that too)?